modules/nf-core/gatk4/preprocessintervals/meta.yml

```diff
@@ -0,0 +1,59 @@
+name: "gatk4_preprocessintervals"
+description: Prepares bins for coverage collection.
+keywords:
+  - gatk4
+  - preprocessintervals
+  - interval
+  - bed
+tools:
+  - "gatk4":
+      description:
+        Developed in the Data Sciences Platform at the Broad Institute, the toolkit offers a wide variety of tools
+        with a primary focus on variant discovery and genotyping. Its powerful processing engine
+        and high-performance computing features make it capable of taking on projects of any size.
+      homepage: https://gatk.broadinstitute.org/hc/en-us
+      documentation: https://gatk.broadinstitute.org/hc/en-us/categories/360002369672s
+      doi: "10.1158/1538-7445.AM2017-3590"
+      licence: ["Apache-2.0"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - exclude_intervals:
+      type: file
+      description: Bed file with the genomic regions to be excluded from the analysis (optional)
+      pattern: "*.{bed,interval_list}"
```

Comment:
Can you provide a bed file to this option? Or does it only accept interval_list? 
And if the tool doesn't accept a bed file, then modify the description too.

Reply:
It can take both interval_list files and bed files. I've tested both.

Reply:
👍🏻